src/commands/admin/prune.ts

```diff
@@ -33,43 +28,42 @@ export default class extends Command {
 			new MessageButton({ label: 'Proceed', customId: 'proceed', style: 'DANGER' })
 		];
 
-		const confirmMsg = await msg.channel.send({
+		const confirmMsg = await interaction.channel.send({
 			embeds: [confirmEmbed],
 			components: [new MessageActionRow({ components: confirmBtns })]
 		});
 
-		const collector = msg.channel.createMessageComponentCollector({
+		const collector = interaction.channel.createMessageComponentCollector({
 			filter: i => i.message.id === confirmMsg.id, time: PRUNE_TIMEOUT * 1000
 		});
 
 		const countdown = setInterval(() => this.countdown(confirmMsg, --timeout, confirmBtns, confirmEmbed), 1000);
 
-		collector.on('collect', async interaction => {
-			if (!interaction.isButton()) return;
-			if (interaction.user.id !== msg.author.id) {
-				await interaction.reply({
-					content: 'You cannot respond to a command you did not execute',
+		collector.on('collect', async btnClick => {
+			if (!btnClick.isButton()) return;
+			if (btnClick.user.id !== interaction.user.id) {
+				return await btnClick.reply({
```

Comment:
If you make the change on line 42:
```suggestion
				return await interaction.reply({
```